microsoft-edge/toc.yml

```diff
@@ -24,7 +24,7 @@
 # /top leaf nodes
 # =============================================================================
 # Release notes
-    - name: Release notes
+    - name: Release notes for web platform
```

Comment:
I suggest changing the title to "Release notes for Microsoft Edge web platform".
Can you change this throughout the docs?

For example the title in `/web-platform/release-notes/index.md` should be changed from "Release Notes for web platform for Microsoft Edge" to "Release notes for Microsoft Edge web platform".

I think this works better.

Reply:
toc bucket name "Release notes for Microsoft Edge web platform" might be long, but that could be justified since this PR expands the function of this new page to be a guide to all What's New's & RelNotes in the docset.
@captainbrosset

Reply:
result:
```
    - name: Release notes for Microsoft Edge web platform
      items:
        - name: Release notes for Microsoft Edge web platform
          href: ./web-platform/release-notes/index.md
          displayName: what's new, announcements
```